### PR TITLE
Added css styling to override new default input range styling

### PR DESCRIPTION
**Pull request by StephenArg**

### CODE REVIEW

**Comment by Zarel** [comment]
Hi, I haven't forgotten about this and will get around to it soon. Just posting an update so you know.

**Comment by StephenArg** [comment]
Thanks, I appreciate that!

File: style/client.css [hunk 1]
```diff
@@ -3092,7 +3092,38 @@ a.ilink.yours {
 	height: 16px;
 	font-size: 9pt;
 }
-
+.selector:not(*:root), input[type=range]{
```
Comment (Zarel) [inline style/client.css:3095]: I can't tell what `.selector` is doing here. We don't seem to have any element with `class="selector"`?

Comment (StephenArg) [reply style/client.css:3095]: I noticed my changes were breaking on Firefox (and potentially other browsers). This snippet makes it only work on Chrome. I'd be lying if I said I understand what it's doing exactly, I found it on stackoverflow or something. It works though.

Comment (Zarel) [reply style/client.css:3095]: Hm, I'll try to address Firefox separately.

Comment (Zarel) [reply style/client.css:3095]: Thanks for explaining it to me! I ended up going with a more readable workaround for Firefox: df710473067d3856fa4

**Pull request closed without merging**